### DISCUSSION TITLE
Request for help setting up and troubleshooting the " Freeplane_UtilityPanels " plugin

### DISCUSSION THREAD

**Discussion by aaa1386** [freeplane/freeplane#2154] [Help]
There is a plugin for FP at https://github.com/euu2021/Freeplane_UtilityPanels.
I am having problems setting it up and getting it to work properly. Can anyone help me?

**Comment by aaa1386** [comment]
I have some problems using utilityPanels
As you can see in the image above, the dimensions of this script are not proportional to my computer. I made some changes that made things better for me, but I still have some problems.

One of the problems is that I use a right-to-left layout because the language I use is right-to-left, but this script is completely based on the left-to-right language. Is it possible to make changes to the utilityPanels.groovy file so that the display is completely right-to-left?


Another problem is that the Quick Search panel option. is not working. However, the jumping filter works.
When I start typing in the Quick Search panel, it gives an error and no results are displayed. I send the error file.


[bug.zip](https://github.com/user-attachments/files/18149505/bug.zip)


This is a picture of the version I use
![image](https://github.com/user-attachments/assets/8434f0a6-2e93-4570-a5f0-40fdebf130c5)


![image](https://github.com/user-attachments/assets/5ae7d3f5-5b87-4cbe-a5a5-ed863cfb8a75)

**Comment by HaaiHenkie** [comment]
Hi Alij,

That script has its own repository. I suggest you post a question there, so you directly address the maker of the script. There is also a recent issue posted by quickfold, saying that he has problems launching utility panels. It might be the same problem.

Cheers,

Henk

**Reply by aaa1386** [reply]
>

There is no option to post a comment on the page where the script is written.
I don't know where to raise this issue.

**Reply by aaa1386** [reply]
>
> 
> That script has its own repository. I suggest you post a question there, so you directly address the maker of the script. There is also a recent issue posted by quickfold, saying that he has problems launching utility panels. It might be the same problem.
> 
Can you send the relevant link?

**Reply by HaaiHenkie** [reply]
It is the link in your first message!

**Reply by HaaiHenkie** [reply]
To spell it out: with the link you provided you end up on the "Code" tab, the first tab. Click the second tab, "Issues", and you will see issue 2 submitted by quickfold. Check if it is similar to your findings. If not, you could submit a new issue.

This repository does not have a "Discussions" tab like Freeplane.

**Reply by aaa1386** [reply]
> To spell it....
Thank you very much - I have now written two good ideas on the page where the script was created. You can also give your opinion.

**Comment by euu2021** [comment]
This bug was reported at https://github.com/euu2021/Freeplane_UtilityPanels/issues/2

I will try to help there.
